plone/recipe/varnish/templates/start_script.jinja2

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+exec {{daemon}} \
+{% if version == '4.0' %}
+{% if user %}
+    -p user={{user}} \
+{% endif %}
+{% if group %}
+    -p group={{group}} \
+{% endif %}
+{% endif %}
+{% if version in ('4', '4.1') %}
+{% if user %}
+    -j unix,user={{user}}{% if group %},ccgroup={{group}}{% endif %} \
+{% endif %}
+{% endif %}
+    -f "{{cfg_file}}" \
+    -P "{{pid_file}}" \
+    -a {{bind}} \
+{% if cache_type == 'malloc' %}
+    -s {{cache_type}},{{cache_size}} \
+{% else %}
+    -s {{cache_type}},"{{cache_location}}",{{cache_size}} \
```

Comment:
It looks like you could simplify this by always printing `cache_type` and `cache_size` and conditionally adding `cache_location`.